src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -77,11 +77,14 @@ def start(scenario, settings, events, aggregator_device_mapping, saved_state):
 
         if events is not None:
             events = ast.literal_eval(events)
+        GlobalConfig.start_date = instance((datetime.combine(date.today(), datetime.min.time()))) \
+            if d3a.constants.IS_CANARY_NETWORK else GlobalConfig.start_date
 
         config_settings = {
             "start_date":
                 instance(datetime.combine(settings.get('start_date'), datetime.min.time()))
-                if 'start_date' in settings else GlobalConfig.start_date,
+                if 'start_date' in settings and d3a.constants.IS_CANARY_NETWORK is False
+                else GlobalConfig.start_date,
```

Comment:
That's way too dangerous for now. I would recommend leaving everything as it was, and only doing `config_settings["start_date"] = date.today()` after having created the config_settings (line 110 of this file)